tests/utilities.mjs

```diff
@@ -0,0 +1,17 @@
+export function isVisible(element) {
```

Comment:
add comment about why need that. something that `jest-expect` do out of the box and `node:assert` didn't do